### PR TITLE
Update the 1.x version of this plugin to work with dash.js 2.0.0

### PR DESCRIPTION
**Pull request by forbesjo**
This PR updates the 1.x branch (compatible with video.js 4.x) to use dash.js 2.0.0

Todo:
- [x] fix the tests

### CODE REVIEW

**Comment by forbesjo** [comment]
@imbcmdth could you take a look at this when you get a chance?

File: package.json [hunk 1]
```diff
@@ -25,19 +25,25 @@
...
     "grunt-contrib-uglify": "~0.2.0",
     "grunt-contrib-watch": "~0.4.0",
+    "grunt-karma": "^0.12.1",
+    "karma": "^0.13.15",
+    "karma-browserstack-launcher": "^0.1.9",
+    "karma-chrome-launcher": "^0.2.2",
+    "karma-detect-browsers": "^2.0.2",
+    "karma-firefox-launcher": "^0.1.7",
+    "karma-qunit": "^0.1.8",
+    "karma-safari-launcher": "^0.1.1",
```
Comment (gesinger) [inline package.json]: Do we need the safari launcher?

File: test/integration.test.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  });
+
+  q.test('should play', function(assert) {
+    var
+      done = assert.async(),
+      player = this.player;
+
+    player.one('seeked', function() {
+      player.one('timeupdate', function() {
+        assert.ok(true, 'seeked to 15s and continued playing');
```
Comment (gesinger) [inline test/integration.test.js]: Do we want to verify that the time is around 15s?

File: package.json (hunk 1 above)
Comment (forbesjo) [inline package.json]: I'll remove it since we don't use it in this project.

File: test/dashjs.test.js [hunk 3]
```diff
@@ -52,66 +52,59 @@
...
-            strictEqual(resetSrcCalled, true, 'Html5DashJS#resetSrc_ was called');
-
-            tech.dispose();
-
-            // Restore
-            Dash.di.DashContext = origContext;
-            window.MediaPlayer = origMediaPlayer;
-            videojs.xhr = origVJSXHR;
-            videojs.Html5DashJS.prototype.resetSrc_ = origResetSrc;
+          create : function () {
```
Comment (gesinger) [inline test/dashjs.test.js]: Extra space after create

**Comment by gesinger** [comment]
LGTM

**Comment by gesinger** [comment]
LGTM

**Pull request merged**